packages/mobile/src/send/reducers.ts

```diff
@@ -48,13 +56,25 @@ export const sendReducer = (
   }
 }
 
+const sendPaymentOrInvite = (state: State, amount: BigNumber) => {
+  const latestPayment = { timestamp: Date.now(), amount }
```

Comment:
Before we talked about  using the payment success action. otherwise failed unsent payments count against your told which is not so fun. 

also you were gonna pass timestamp in to the action from the saga.

Reply:
Yes, I am still not completely sure about the best place but here is my thinking, after discussing a bit with Connor also. If we store in payment success only then someone in theory could go into airplane mode, then enqueue a bunch of transactions and then turn airplane off. That would allow him to go over the limit.

This is why I think doing it in sendPayment, and then removing failed transactions (I still need to implement that) is a bit better. But, I am open to suggestions of course.